src/link/Elf.zig

```diff
@@ -489,6 +507,29 @@ pub fn populateMissingMetadata(self: *Elf) !void {
         });
         self.phdr_table_dirty = true;
     }
+    if (self.phdr_load_ro_index == null) {
+        self.phdr_load_ro_index = @intCast(u16, self.program_headers.items.len);
+        self.rodata_block_list.phdr_index = self.phdr_load_ro_index;
+        // TODO Find a hint about how much data need to be in rodata ?
+        const file_size = 1024;
+        // Same reason as for GOT
+        const p_align = if (self.base.options.target.os.tag == .linux) 0x1000 else @as(u16, ptr_size);
+        const off = self.findFreeSpace(file_size, p_align);
+        log.debug("found PT_LOAD free space 0x{x} to 0x{x}\n", .{ off, off + file_size });
```

Comment:
Log automatically inserts a "\n" so you don't need another.

Reply:
All the other `log.debug` lines in that file also have the "\n", so "\n" here is fine too.

Reply:
It is a mistake https://discord.com/channels/605571803288698900/785499283368706060/818987635883114547 
```
[6:24 PM]
andrewrk:
the \n is a mistake in all the logs
[6:24 PM]
andrewrk:
if you copied it from me then you copied my typo :slight_smile:
```

Reply:
I removed the "\n" from all the logs.